e2e/tests/example.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { test, expect } from '@playwright/test';
+
+test('has title', async ({ page }) => {
+  await page.goto('https://playwright.dev/');
```

Comment:
Hi @luciemars thanks for starting this!
Do you know if i can happily serve a local file here for my test setup or if i have to run a webserver?

Reply:
I found this in the docs https://playwright.dev/docs/test-advanced#launching-a-development-web-server-during-the-tests.
I can look a bit more if you want to do it differently.

Reply:
no no i can look that myself no worries, i was just wondering if you had an answer.
I want to add a real fabricJS test before merging, i ll keep you posted